src/app/store.ts

```diff
@@ -1,13 +1,19 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersReducer, { fetchUsers } from '../features/users/usersSlice';
+import postsReducer from '../features/posts/postSlice';
 
 export const store = configureStore({
   reducer: {
     counter: counterReducer,
+    users: usersReducer,
+    posts: postsReducer,
```

Comment:
Verify that the `usersReducer` and `postsReducer` are correctly handling their respective states and actions. This will ensure the store behaves as expected.